src/cdk-experimental/testing/test-element.ts

```diff
@@ -101,4 +101,11 @@ export interface TestElement {
 
   /** Gets the dimensions of the element. */
   getDimensions(): Promise<ElementDimensions>;
+
+  /**
+   * Flushes change detection and async tasks.
+   * In most cases it should not be necessary to call this. However, there may be some edge cases
+   * where it is needed to fully flush animation events.
+   */
+  stabilize(): Promise<void>;
```

Comment:
Any naming preference? I could also see `flush`

Reply:
Maybe `whenStable` to match the one on the `ComponentFixture`? I don't have a strong preference either.

Reply:
We chatted and Miles suggested `forceStabilize` which I really like because it has a built in discouragement from using when its not necessary